README.md

```diff
@@ -51,6 +51,21 @@ $ kubectl -n dynatrace logs -f deployment/dynatrace-oneagent-operator
 #### OpenShift
 ```sh
 $ oc adm new-project --node-selector="" dynatrace
+```
+
+If you are installing the Operator on an **OpenShift Container Platform 3.11** environment, in order to use the certified [OneAgent Operator](https://access.redhat.com/containers/#/registry.connect.redhat.com/dynatrace/dynatrace-oneagent-operator) and [OneAgent](https://access.redhat.com/containers/#/registry.connect.redhat.com/dynatrace/oneagent) images from [Red Hat Container Catalog](https://access.redhat.com/containers/) (RHCC), you need to [provide image pull secrets](https://access.redhat.com/documentation/en-us/openshift_container_platform/3.9/html/developer_guide/dev-guide-managing-images#pulling-private-registries-delegated-auth). The Service Accounts on the `openshift.yaml` manifest already have links to the secrets to be created below.
```

Comment:
I think you could say say this more concisely, and I am not sure the links are necessary. I like short bits of text with fewer words.

https://www.evernote.com/l/AIAfY6BeAJtCMZzyVeCGM9mfGJ3ENZIiJZQ